parsers/ROI.py

```diff
@@ -0,0 +1,52 @@
+"""Parser for the Republic Of Ireland."""
+import logging
+from datetime import date, datetime, time, timedelta
+
+import pandas as pd
+
+import arrow
+import requests
+
+
+QUERY_ROOT = f'http://smartgriddashboard.eirgrid.com/DashboardService.svc/data?'
+
+MAPPING = {'FUEL_COAL': 'coal',
+           'FUEL_GAS': 'gas',
+           'FUEL_RENEW': 'renew',
```

Comment:
I'm not sure if the backend accepts 'renew'.

Reply:
Indeed, should it go as 'unknown' then ?

Reply:
ENTSO-E separates renewables. https://www.electricitymap.org/zone/IE